### PR TITLE
package.json updated

### LINKED ISSUE #779: node_modules/hoodie folder is 23,8mb
image

nearly all of it are unneeded things like the pandoc binary and docs. We should remove it by adding the ["files" setting](https://docs.npmjs.com/files/package.json#files) to `hoodie`’s package.json file and only list what’s needed to be published

### PR DESCRIPTION
**Pull request by sohini-roy**
closes #779

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -53,6 +50,7 @@
   "engines": {
     "node": ">=4"
   },
+  "files":["@hoodie/admin", "@hoodie/client", "@hoodie/server"],
```
Comment (gr2m) [inline package.json]: the `"files"` key in the `package.json` is not for names of dependencies, but for file names and paths that are part of the repository, please read the description at that’s not how you use `files`. See the documentation at https://docs.npmjs.com/files/package.json#files

**Review by gr2m — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (gr2m) [reply package.json]: e.g. see https://github.com/hoodiehq/hoodie-admin/blob/master/package.json#L14-L16. This means that when the package get published to the npm registry, only the contents of the `dist/` folder get uploaded, everything else is ignored. Does that  make sense

Comment (sohini-roy) [reply package.json]: i did follow that. i am sure i misunderstood but i am still confused.

Comment (gr2m) [reply package.json]: when you install `hoodie` like it’s described in the [quickstart guide](http://docs.hood.ie/en/latest/guides/quickstart.html) and then open `node_modules/hoodie`, you will see a ton of files in there, including the pandoc file which is 20mb of size. Hoodie doesn’t need that file to work, nor does it need e.g. the docs folder. The only file it needs are the JavaScript files that get executed when you run `npm start` in your app. These are `bin/`, `cli/` and `server/`. And then it also needs the `public/` folder as it has the default UI that will be shown when your app has no `own` public folder. So the names of these 4 folders are the one that we need to to set `"files"` to in our `package.json`. That way only these folders will be uploaded to the npm registry, all other files will be ignored. Does that make sense?

Comment (sohini-roy) [reply package.json]: yes i get it

**Comment by sohini-roy** [comment]
please have a look and review

File: package.json [hunk 2]
```diff
@@ -11,9 +11,6 @@
     "url": "https://github.com/hoodiehq/hoodie/issues"
   },
   "dependencies": {
-    "@hoodie/admin": "^1.0.1",
-    "@hoodie/client": "^10.0.0",
-    "@hoodie/server": "^23.0.0",
```
Comment (gr2m) [inline package.json]: you need to put these back in, that’s why the tests are currently failing, see https://travis-ci.org/hoodiehq/hoodie/builds/249421274#L564

File: package.json [hunk 3]
```diff
@@ -53,6 +50,9 @@
   "engines": {
     "node": ">=4"
   },
+  "files": [
+    "bin", "cli", "public", "server"
+  ],
```
Comment (gr2m) [inline package.json:58]: that looks good 👍

**Review by gr2m — CHANGES_REQUESTED** [review]

File: package.json (hunk 3 above)
Comment (sohini-roy) [reply package.json:58]: thanks for helping :)

**Review by gr2m — APPROVED** [review]
great, thanks :)

**Comment by gr2m** [comment]
and released :) https://github.com/hoodiehq/hoodie/releases/tag/v28.2.1

**Pull request merged**